en/style/typography.md

```diff
@@ -30,42 +34,64 @@ These styles also determine the type of text element and typography class that w
 
 > [!Warning]
-> In Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
+> In Figma and Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
 
 ## Component Specific Typography
 
 Components, such as [Avatar](../components/avatar.md), [Button](../components/button.md), [Grid](../components/grid.md), [Slider](../components/slider.md) and [Tooltip](../components/tooltip.md) use component-specific Typography to accommodate the specifics of the styling used by the respective component e.g. the Avatar with initials and Button use an all caps style. Below you can find the text styles defined for the Avatar in Sketch to allow more versatile usage of font colors.
 
-<img class="responsive-img" src="../images/typography_colors.png"/>
+<img class="responsive-img" src="../images/typography_colors.png" srcset="../images/typography_colors@2x.png 2x"/>
 
-## Typography Color Variants in Sketch
+## Typography Color Variants
 
-In Sketch, the text styles include the text color, thus each one comes in multiple preset colors, such as `active` corresponding to `grays.900`, `inactive` corresponding to `grays.700`, `disabled` corresponding to `grays.500`, `white` and `secondary`. There are also scales with special colors for strings that need to be in `success`, `warn`, and `error` colors, as well as some additional nuances used by the Indigo.Design Components.
+- In **Sketch**, the text styles include the text color, thus each one comes in multiple preset colors, such as `active` corresponding to `grays.900`, `inactive` corresponding to `grays.700`, `disabled` corresponding to `grays.500`, `white` and `secondary`. There are also scales with special colors for strings that need to be in `success`, `warn`, and `error` colors, as well as some additional nuances used by the Indigo.Design Components.
 
-<img class="responsive-img" src="../images/typography_base_colors.png"/>
+<img class="responsive-img" src="../images/typography_base_colors.png" srcset="../images/typography_base_colors@2x.png 2x"/>
+<div class="divider--half"></div>
 
-## Changing the Typeface in Sketch
+- In **Figma** and **Adobe XD** the text styles do not include the text color, and you can adjust it based on the design needs. However, the Text styles applied in all the Indigo.Design system library components are aligned to these color variants mentioned above.
```

Comment:
I guess now that Figma library is the most up-to-date, maybe not all styles match the predefined color styles in Sketch, but maybe that's not relevant. @sdimchevski